src/computer.c

```diff
@@ -102,13 +108,18 @@ enum OperationReturnCode ld_del_computer(LDHandle *handle, const char *name, con
  */
 enum OperationReturnCode ld_mod_computer(LDHandle *handle, const char *name, const char *parent, LDAPAttribute_t **computer_attrs)
 {
-    TALLOC_CTX *talloc_ctx = talloc_new(NULL);
+    TALLOC_CTX *talloc_ctx;
```

Comment:
Consider using `TALLOC_CTX *talloc_ctx = NULL;` instead of `TALLOC_CTX *talloc_ctx;`.